corehq/apps/receiverwrapper/util.py

```diff
@@ -74,7 +74,7 @@ def get_meta_appversion_text(form_metadata):
         return None
 
 
-@quickcache(['domain', 'build_id'], timeout=24*60*60)
+@quickcache(["domain", "build_id"], timeout=24 * 60 * 60)
```

Comment:
The quote changes were not required to fix a lint error nor done by isort, right? Is double-quote instead of single- a personal style preference?

Reply:
Good catch! I formatted that line with black and didn't notice the quote change (I should have). Happy to change it back if you'd prefer.

Reply:
Offlined wiht Daniel and there is no preference on the quotes, just was a question out of curiosity.